client/src/controllers/TabsController.test.js

```diff
@@ -0,0 +1,256 @@
+import { Application } from '@hotwired/stimulus';
+import { TabsController } from './TabsController';
```

Comment:
```suggestion

import { TabsController } from './TabsController';
```

Small nitpick, adding an extra line break here.